### DISCUSSION TITLE
Option to create multiple instances of a data object by specifying the count (e.g., 2)

### DISCUSSION THREAD

**Discussion by danielh-official** [spatie/laravel-data#802] [Ideas]
This can be an optional argument in the `toArray` and `all` functions.

Just like how Laravel Factories have a "count" option that can be specified in the `factory` method (e.g., `Model::factory(10)->create()` will create 10 models).

Essentially, when creating data objects, I would set the default values to faker variables to imitate the factory structure and then set the resulting object to array.

### Current Setup

```
$data = [Data::toArray(), Data::toArray()]
```

### Proposed Setup

```
$data = Data::toArray(2)
```

**Comment by rubenvanassche** [comment]
That would technically not be possible since it would violate the Arrayable interface.
